.github/workflows/test.yml

```diff
@@ -7,7 +7,6 @@ jobs:
     runs-on: ubuntu-18.04
     strategy:
       matrix:
-        # LCG: ["LCG_96b/x86_64-centos7-gcc9-opt","LCG_96b/x86_64-centos7-gcc8-opt","LCG_96b/x86_64-centos7-clang8-opt","LCG_96bpython3/x86_64-centos7-gcc8-opt"]
         LCG: ["LCG_96b/x86_64-centos7-gcc9-opt"]        
```

Comment:
is this LCG still used any where?

Reply:
Actually no, and we are not testing with different compilers either, so I will remove it.